src/test/java/org/folio/edge/sip2/handlers/CheckoutHandlerTests.java

```diff
@@ -86,6 +87,61 @@ public void canExecuteASampleCheckoutUsingHandler(
         })));
   }
 
+  @Test
+  void cantExecuteASampleCheckoutUsingHandler(
+      @Mock CirculationRepository mockCirculationRepository,
+      Vertx vertx,
+      VertxTestContext testContext) {
```

Comment:
Sorry now I'm obsessing about spaces. Does this block need to come forward 1 space?